examples/tensorflow_hello_world/helloworld.py

```diff
@@ -9,91 +9,106 @@
 from xain.fl.coordinator.aggregate import FederatedAveragingAgg
 from xain.fl.participant import ModelProvider, Participant
 
-# TODO: this feels a bit hacky. it's the only flag that needs to be added here.
-#  any better solution?
+# Defining the 'task_name' flag here, to be used by the absl-py app.
 FLAGS = flags.FLAGS
 flags.DEFINE_string("task_name", None, "")
 
 
-# We first define some constants for this specific example
+DATASET_NAME = "fashion-mnist-100p-iid-balanced"
```

Comment:
Reading it now it seems like we can inline all single-use constants, i.e. remove the definition here and add the value (+ documentation) where it is actually being used.

Reply:
uhm i think it's important the user can familiarise herself with the federated hyper-parameters and has a centralised place where to set those parameters, instead of inline.